components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/resources/RoleResourceV2.java

```diff
@@ -162,7 +164,8 @@ public Response getRoles(@HeaderParam(SCIMProviderConstants.ACCEPT_HEADER) Strin
             // Create charon-SCIM role endpoint and hand-over the request.
             RoleResourceV2Manager roleResourceManager = new RoleResourceV2Manager();
             SCIMResponse scimResponse = roleResourceManager
-                    .listWithGETRole(roleManager, filter, startIndex, count, sortBy, sortOrder);
+                    .listWithGETRole(roleManager, filter, startIndex, count, sortBy, sortOrder, attribute,
```

Comment:
shouldn't a similar change be done for getRolesByPOST ?

Reply:
No, in the 'getRolesByPOST' method, attributes and excludeAttributes are sent in the request body, not as request parameters.